### ISSUE TITLE
colons (:) not rendered as literal in titles

### ISSUE THREAD

**Issue by andy5995** [ismnoiet/jekyll-posts-generator#2]
I ran `jposts` to create a jekyll post that had a `:` in the title, but although I used quotes in the title, the colon appeared as `o` in the post title and wrapped the line in the post file.

I see you haven't worked on this project for over a year, but thought I'd file a ticket in case you ever worked on this project again.

I think my team and I will be able to use `jposts` for our new project (so thank you!) but we may have to do a bit of hacking on it. ;) (mhwkb/mhwkb.github.io#4)

**Comment by ismnoiet** [comment]
Thank you @andy5995 for raising the issue, I will try to reproduce the issue and submit the fix,
but i don't mind if you can submit a PR regarding this issue :), it would be more than welcomed.

**Comment by andy5995** [comment]
I tried your program again today and didn't have the same problem. I'll keep an eye on it though and if I can reproduce it every time I'll report it here.

Thanks for following up. This program is a great time-saver!

**Comment by ismnoiet** [comment]
You're welcome @andy5995.

**Comment by ismnoiet** [comment]
@andy5995 I tried it and it is working correctly,
example:
```
jposts create-post "tmp hello:there"
```
I close this issue, if you still have the problem please comment and give what you are using as input.

**Comment by andy5995** [comment]
I couldn't reproduce it either.  I'll be sure to note all the details if it happens again.

Would you be interested in doing a PR and adding your project files to my repo in this directory?

https://github.com/mhwkb/mhwkb.github.io/tree/master/post_generators

If so, the README should contain a link back to your project page noting that you're the author.

**Comment by ismnoiet** [comment]
@andy5995 To be honest my time is very limited, may be in the future.
Thank you for the nice suggestion.

**Comment by andy5995** [comment]
You're welcome.

I decided it doesn't make much sense to have the programs in my repo, and will remove the one that's been added.

If you ever go back to work on jekyll-posts-generator, let me know and I'll add your link to mhwkb/mhwkb.github.io#11

You actually inspired a couple of similar programs, and I've added your link to [ACKNOWLEDGEMENTS.md](https://github.com/mhwkb/mhwkb.github.io/blob/master/ACKNOWLEDGEMENTS.md).. so... Thanks!

Best wishes! :)
